src/__tests__/ButtonToggle.spec.js

```diff
@@ -10,12 +10,6 @@ describe("ButtonToggle", () => {
     expect(wrapper.first(Button).shallow().text()).toBe("Ello world");
   });
 
-  it('should have button already toggled for defaultValue true', () => {
```

Comment:
Why was this removed?

Reply:
This was removed since the state can be accessed only inside class components. Should we add `react-testing-library` to solve this, or are there any other suggestions?

Reply:
You can do this with enzyme by:
```
wrapper.find(Button).props().active
```
and run expectations on that.

Although I do recommend going with react-testing-library instead as enzyme is technically a dead project now.